plugin/src/Reconciler.lua

```diff
@@ -53,11 +77,9 @@ function Reconciler:reconcile(virtualInstancesById, id, instance)
 	self.instanceMap:insert(id, instance)
 
 	-- Some instances don't like being named, even if their name already matches
-	setCanonicalProperty(instance, "Name", virtualInstance.Name)
-
-	for key, value in pairs(virtualInstance.Properties) do
-		setCanonicalProperty(instance, key, rojoValueToRobloxValue(value))
-	end
+	writeInstance(instance, {
+		Name = virtualInstance.Name,
+	})
```

Comment:
We should probably have an API that writes a single property instead of using this general purpose one for names and stuff

Reply:
So write an inline function in the Reconciler or this should be in rbx_dom_lua?

Reply:
Probably something in rbx\_dom\_lua. I'd like to revisit these APIs before we dive so deep into trying to integrate them into Rojo.

Reply:
I've sent a PR for this on rbx_dom_lua a bit ago.